airbyte-container-orchestrator/src/main/java/io/airbyte/container_orchestrator/config/ContainerOrchestratorFactory.java

```diff
@@ -85,6 +85,7 @@ ProcessFactory kubeProcessFactory(
     return new KubeProcessFactory(
         workerConfigsProvider,
         configs.getJobKubeNamespace(),
+        configs.getJobKubeServiceAccount(),
```

Comment:
This is a pretty minor thing, but we're eventually seeking to deprecate the `Configs` and `EnvConfigs` approach in favor of direct Micronaut injection. So instead of defining a new `Configs` method at all, it would be preferable to inject the service account just like you're doing in the `ProcessFactoryBeanFactory` class, ie `@Value("${airbyte.worker.job.kube.serviceAccount}") final String serviceAccount`

Reply:
update: I made this change on our internal repo